### PR TITLE
Fix avatar billboards all rendering as the same billboard

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

File: interface/src/avatar/Avatar.cpp [hunk 1]
```diff
@@ -537,18 +537,13 @@ void Avatar::renderBillboard() {
...
-        _billboardTexture.reset(new Texture());
-        glBindTexture(GL_TEXTURE_2D, _billboardTexture->getID());
-        glTexImage2D(GL_TEXTURE_2D, 0, GL_RGBA, image.width(), image.height(), 0,
-            GL_BGRA, GL_UNSIGNED_BYTE, image.constBits());
-        glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
-    
-    } else {
-        glBindTexture(GL_TEXTURE_2D, _billboardTexture->getID());
+        // Using a unique URL ensures we don't get another avatar's texture from TextureCache
+        QUrl uniqueUrl = QUrl(QUuid::createUuid().toString());
```
Comment (huffman) [inline interface/src/avatar/Avatar.cpp]: Ideally `ResourceCache`/`TextureCache` would provide a way to not use cached resources, or another way to generate `NetworkTexture` objects that don't need to be tied to the `TextureCache`.

**Comment by hifi-gustavo** [comment]
OS X Yosemite functional test completed with errors
Download recording here:
https://s3.amazonaws.com/hifi-public/pr-builds/recordings/4319.mov

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5c3cae471be2e0527abfe8eea8ca8a84193c8a6f available at: https://gist.github.com/d73e1c02d60eb46497db<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4319.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4319.dmg

**Comment by huffman** [comment]
Anyone interested in taking a look at this? :smiley:

**Pull request merged**